### PR TITLE
feat(ktooltip): introduce kui tokens [KHCP-7730]

### PR DESCRIPTION
**Pull request by mptap**
# Summary

https://konghq.atlassian.net/browse/KHCP-7730

**Changes:**

- introduces `kui-` design tokens in `KTooltip` component
- removes any usage of Kongponents utility classes in `KTooltip` component template

## PR Checklist

* [ ] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/kongponents#committing-changes).
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: docs/components/tooltip.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ KTooltip won't be triggered if the element you pass through `default` slot has `
 
 <KCard>
   <template #body>
-    <div class="d-flex">
-      <KTooltip label="I won't pop up" class="mr-2">
+    <div class="k-tooltip">
```
Comment (portikM) [inline docs/components/tooltip.md:39]: Could you not use `k-tooltip` class in the docs? Make it something more generic, like maybe `my-tooltip`  so it doesn't overlap with component styles. Same for `k-tooltip-label`

**Review by portikM — CHANGES_REQUESTED** [review]
Could you remove `color-red-500` from usage in this file? Maybe replace the whole tag with something like `<em>`

File: docs/components/tooltip.md (hunk 1 above)
Comment (mptap) [reply docs/components/tooltip.md]: updated all occurrences to generic lass names

**Review by portikM — APPROVED** [review]
LGTM 🚀

**Review by jillztom — APPROVED** [review]

**Pull request merged**